lib/charms/hpc_libs/v0/slurm_ops.py

```diff
@@ -584,6 +595,42 @@ def _env_manager_for(self, type: _ServiceType) -> _EnvManager:
         return _EnvManager(file=self._env_file, prefix=type.value)
 
 
+class _JWTKeyManager:
+    """Control the JWT key used by Slurm.
+
+    Todo:
+        Use `jwtctl` to provide backend for generating, setting, and getting
+        JWT key used by `slurmctld` and `slurmrestd`. This way we won't need to
+        pass the `snap` bool as an argument to `__init__`
```

Comment:
Private classes like `_JWTKeyManager` are not rendered as part of the public documentation for [`slurm_ops`](https://charmhub.io/hpc-libs/libraries/slurm_ops) on Charmhub; only the docstrings for public classes/functions are shown for charm libraries.

However, I think your comment here indicates that we need an agreeable standard/practice for having TODO comments in our code. As our portfolio grows, so will the times where we'll need to implement short-term solutions that "work" but aren't how we want it to be in the future. _I'm thinking about our discussion from earlier today about how we need to spec out the best-practices we follow._

Since we already follow the Google docstring format, perhaps we also follow the TODO styleguide as well? https://google.github.io/styleguide/pyguide.html#312-todo-comments. Let me know what you think @jedel1043 :nerd_face:

Reply:
Sounds good to me!